aten/src/ATen/native/native_functions.yaml

```diff
@@ -2577,6 +2577,12 @@
   dispatch:
     CPU: per_tensor_affine_qtensor_cpu
 
+- func: set_storage_(Tensor(a!) self, Storage storage, int storage_offset, int[] sizes, int[] strides=[]) -> Tensor(a!)
```

Comment:
wait, how does this end up generating qtensor_set_storage_, I don't see that string anywhere?

but having a different method isn't the best API -- can't you either specify different dispatch here or in the function that implements set_?  (check with @li-roy).

Reply:
I changed that to `set_storage_` since it works for all Tensors